rpcserver.go

```diff
@@ -6101,7 +6101,7 @@ func (r *rpcServer) UpdateChannelPolicy(ctx context.Context,
 
 	// With the scope resolved, we'll now send this to the local channel
 	// manager so it can propagate the new policy for our target channel(s).
-	err := r.server.localChanMgr.UpdatePolicy(chanPolicy, targetChans...)
+	err := r.server.localChanMgr.UpdatePolicy(chanPolicy, false, targetChans...)
```

Comment:
As is, this would _always_ override the default channel enable/disable state, setting it to false.